### DISCUSSION TITLE
I can't launch NeoForge game through TLauncher

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [neoforged/NeoForge#811] [General]
Hello, I have a problem.

I had problems with the normal Minecraft Launcher a few days ago (problems with opening Minecraft Launcher & problems related to Microsoft Store), so I switched to TLauncher instead. I don't want to make a new issue, so I created a discussion instead.

I tried to launch vanilla 1.20.4, and it worked perfectly. But if I tried NeoForge-20.4.232, then I couldn't launch the game.

I don't know what is the problem, but here are things, that I already tried:
- Delete the ".minecraft" folder & re-installed TLauncher
- Delete the "assets" folder in ".minecraft"
- Try to change the Java version (i.e. jdk-21, jdk-22, jre-1.8)
- Update symbolic links (symlinks) in C:\programdata\Oracle\Java\javapath
- Change TLauncher RAM allocation

Furthermore, I'm attaching the latest crash log file:
[tlauncher_04_16_2024_15_34_51.log](https://github.com/neoforged/NeoForge/files/14998023/tlauncher_04_16_2024_15_34_51.log)

Thank you for any help.

**Comment by AterAnimAvis** [comment]
We don't provide support for TLauncher.

**Reply by Aleksandra0056** [reply]
Why? Why you just close my ticket? Can you stop censoring other people's posts?

**Reply by sh0inx** [reply]
They very clearly stated that they don't support TLauncher, which is why the discussion was closed. You can try using something like [PrismLauncher](https://prismlauncher.org/) or the [Modrinth App](https://www.modrinth.com/app), but it is up to the launcher to properly launch the game and support their users.

**Reply by AterAnimAvis** [reply]
Our support for third party launchers is limited, we target the vanilla launcher, whilst we will help where we can we don't know the specifics for each individual launcher.
Saying that, we do not provide support for TLauncher or other Launchers that allow piracy, at all.

**Reply by Aleksandra0056** [reply]
Screw you man!

**Reply by sh0inx** [reply]
well thats not very nice

**Reply by Aleksandra0056** [reply]
Stop censoring other people's posts! I don't care if it allows piracy, just reopen it!
